.gitignore

```diff
@@ -24,3 +24,4 @@ Skype.framework
 Podfile.lock
 Pods
 *.xcworkspace
+src/client/AM.PA.MacOs/PersonalAnalytics.xcodeproj/xcuserdata/luigiquaranta.xcuserdatad/
```

Comment:
This should be changed to not include your username.
Maybe `src/client/AM.PA.MacOs/PersonalAnalytics.xcodeproj/` is enough?